### PR TITLE
[Term Entry] Kotlin `.log()` method

### LINKED ISSUE #3527: [Term Entry] Kotlin Math .log()
### Reason/inspiration (optional)

We would like a new term entry on the `.log()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/log/log.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by manviii27**
### Description

A new term entry for Kotlin `.log()` method.
Close #3527 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+Title: 'log()'
+Description: 'Calculates the logarithm of a number with respect to base.'
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/log/log.md:3]:
```suggestion
Description: 'Calculates the logarithm of a number with respect to a base value.'

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.log()`** method in Kotlin's `math` class is used to calculate the logarithm of a number with respect to base.
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/log/log.md:15]:
```suggestion
The **`.log()`** method in Kotlin's `math` class is used to calculate the logarithm of a number with respect to a base value.

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+```
+
+The output of this code will be:
+
+```shell
+3.5849625007211565
+```
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/log/log.md:52]: This needs to be changed to a regular example or removed as codebytes doesn't support Kotlin at this time.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manviii27 I did a first review and left a few comments. Most important is the codebyte needs to be changed or removed because codebytes doesn't support Kotlin.

**Comment by manviii27** [comment]
@SSwiniarski I've made the changes, please review!

**Review by SSwiniarski — APPROVED** [review]
@manviii27 looks good for a second review.

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.log()`** method in Kotlin's `math` class is used to calculate the logarithm of a number with respect to a base value.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/log/log.md:15]:
```suggestion
The **`log()`** method in Kotlin's `math` class is used to calculate the logarithm of a number with respect to a base value.

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Syntax
+
+```pseudo
+log(n, base)
+```
+
+- n: The number of type `double` for which the logarithm is to be calculated.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/log/log.md:23]:
```suggestion
- `n`: The number of type `double` for which the logarithm is to be calculated.

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- base: The number of type `double` that specifies the logarithm base.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/log/log.md:24]:
```suggestion
- `base`: The number of type `double` that specifies the logarithm base.

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+The `.log()` method returns a value of type `double`, the logarithm of `n` to `base`. The following are the special cases:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/log/log.md:26]:
```suggestion
The `log()` method returns a value of type `double`, the logarithm of `n` to `base`. The following values represent special cases:

```

File: content/kotlin/concepts/math-methods/terms/log/log.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+- `log(n, base)` is `NaN` if either `n` or `base` are `NaN`.
+- `log(n, base)` is `NaN` when `n < 0` or `base <= 0` or `base == 1.0`.
+- `log(+Inf, +Inf)` is `NaN`.
+- `log(+Inf, base)` is `+Inf` for `base > 1` and `-Inf` for `base < 1`.
+- `log(0.0, base)` is `-Inf` for `base > 1` and `+Inf` for `base < 1`.
+
+## Example
+
+This example shows how to use the `.log()` method to calculate the logarithm of `12` with base `2`:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/log/log.md:36]:
```suggestion
This example shows how to use the `log()` method to calculate the logarithm of `12` with base `2`:

```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@manviii27 I've added my comments. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by manviii27** [comment]
@caupolicandiaz I've made the changes, please review.

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 @SSwiniarski This is ready to merge

**Pull request merged**